rpc/src/rpc_actor.rs

```diff
@@ -134,6 +150,50 @@ impl Receive<ShellChannelMsg> for RpcServer {
 
     fn receive(&mut self, _ctx: &Context<Self::Msg>, msg: ShellChannelMsg, _sender: Sender) {
         if let ShellChannelMsg::NewCurrentHead(_, block) = msg {
+            // prepare main chain_id
```

Comment:
@bkontur I am thinking that during bootstrap maybe we don't want this, what do you think?

Reply:
@tizoc yes, exactly, I was thinking to move `current_bootstrap_state: SynchronizationBootstrapStateRef,` to RpcEnvironment` there is a is_bootstrapped flag

Reply:
ok, I will add that tomorrow, I need to figure out how (and from where) this is called to add that argument.

Reply:
@bkontur ok, I made this change (the `is_bootsrapped` value was available on the context where this message is produced, so I just used that): https://github.com/tezedge/tezedge/pull/638/commits/d0ffc329df00047d64a24db85f0d82e71831b865